composer.json

```diff
@@ -1,18 +1,26 @@
 {
 	"name": "hm-base",
-	"license": "GPL V3+",
-	"minimum-stability": "stable",
+	"type": "project",
+	"license": "GPL-3.0-or-later",
+	"require": {
+		"php": "7.0.*",
```

Comment:
This would mean that you cannot install when you are running anything other than PHP 7.0.x locally. 7.1 and beyond wouldn't work, So, what we want here is this: `"php": ">=7.0"`, or, if we really want to have a specific patch release as minimum (which should almost never be the case), then we'd have something like `"php": ">=7.0.7"`.

![2018-09-14 10_13_33-mingw64__c_temp_composer-test](https://user-images.githubusercontent.com/6049306/45538287-f1bfe600-b806-11e8-887a-604a9900aed5.png)